### PR TITLE
feat(genesis): creating and querying proposals

### PR DESCRIPTION
**Pull request by ltacker**
Add the transactions to create new `AddAccount` or `AddValidator` proposals for a chain.
Add queries to query the list of pending proposals and show information about a proposal from its proposal ID

We have the commands:
```
spnd q genesis pending-proposals [chain-id]
spnd q genesis show-proposal [chain-id] [proposal-id]
spnd tx genesis proposal-add-account [chain-id] [xtoken1,xtoken2,...] // User who send the message is automatically the account to add
spnd tx genesis proposal-add-validator [chain-id] [peer] [path-to-gentx-file]
```

We currently have some bugs with the `tx` commands above, I'm creating an issue for this but all txs and queries have tests

### CODE REVIEW

**Comment by ltacker** [comment]
CLI issue: #38

File: x/genesis/client/cli/tx.go [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
 	tmjson "github.com/tendermint/tendermint/libs/json"
 	tmtypes "github.com/tendermint/tendermint/types"
-
-	"github.com/spf13/cobra"
+	"io/ioutil"
+	_ "io/ioutil"
```
Comment (ilgooz) [inline x/genesis/client/cli/tx.go:13]: can we perhaps run goimports for the whole source code?

**Review by ilgooz — APPROVED** [review]
Great! :fire:

**Comment by fadeev** [comment]
Great work, @ltacker 🚀

> spnd tx genesis proposal-add-account [chain-id] [xtoken1,xtoken2,...]
> // User who send the message is automatically the account to add

This might work for now, but the purpose of this command is to also be able to add accounts of other, similar to this:

```
gaiad add-genesis-account cosmos1lt6kq4u4me844u858hehu0mwt336tklfpr43mu 1000atom,1000steak
```

**Review by fadeev — APPROVED** [review]

File: x/genesis/client/cli/tx.go (hunk 1 above)
Comment (ltacker) [reply x/genesis/client/cli/tx.go:13]: I did some experimentation on this file, hence those unused import here.
I run it on this file, I think it should be ok for the other sources

**Comment by ltacker** [comment]
> Great work, @ltacker 🚀
> 
> > spnd tx genesis proposal-add-account [chain-id] [xtoken1,xtoken2,...]
> > // User who send the message is automatically the account to add
> 
> This might work for now, but the purpose of this command is to also be able to add accounts of other, similar to this:
> 
> ```
> gaiad add-genesis-account cosmos1lt6kq4u4me844u858hehu0mwt336tklfpr43mu 1000atom,1000steak
> ```

#39

**Pull request merged**